docs/content/references/sui-api/graphql/guides.mdx

```diff
@@ -0,0 +1,257 @@
+---
+title: Guides
+description: Guidelines for working with the Sui GraphQL RPC.
```

Comment:
```suggestion
description: Guidelines for working with the Sui GraphQL RPC, how to build effective queries, and how to migrate from the JSON RPC format.
```
A little context...the `description` is what's shown on search results. The content weighs pretty heavily on SEO results, so want to include Sui GraphQL, JSON RPC, and any other terms you think might help people get to this page when they search for it.